File: app/src/main/java/com/jeksonshar/jweather/repository/Repository.java
package com.jeksonshar.jweather.repository;

import com.jeksonshar.jweather.model.WeatherModel;

import java.util.List;

public interface Repository {

    List<WeatherModel> getAllItemsWeather();

    void saveItemsInRoom(List<WeatherModel> weatherModels);

//    WeatherModel getItemWeatherByID(int id);
//
//    void update(WeatherModel weatherModel); // ??? is need to Jweather?
}

File: app/src/main/java/com/jeksonshar/jweather/repository/Repository.kt
package com.jeksonshar.jweather.repository

import com.jeksonshar.jweather.model.WeatherModel

interface RepositoryDataBase {

    val allItemsWeather: List<WeatherModel>?

    fun saveItemsInRoom(weatherModels: List<WeatherModel>?)

    //    WeatherModel getItemWeatherByID(int id);
    //
    //    void update(WeatherModel weatherModel); // ??? is need to Jweather?
}